.eslintrc.yml

```diff
@@ -3,13 +3,12 @@ extends:
   - semistandard
   - plugin:prettier/recommended
 env:
-  node: yes
-  browser: yes
+  node: true
+  browser: true
 parserOptions:
   ecmaVersion: 5
   ecmaFeatures:
-    globalReturn: no
```

Comment:
Any ideas why this is set? It doesn't seem to be overriding anything in the configs this extends off of (unless I missed something, and it being set results in the `strict` rule enforcing function-level strict mode directives, which I'm assuming is not desirable.

Reply:
I don't know, I haven't understood the meaning of this setting, studying docs doesn't help.
We need this part since we still support IE11.
What is the risk of removing this setting?

Reply:
[Here](https://stackoverflow.com/questions/28955047/why-does-a-module-level-return-statement-work-in-node-js) is an explanation for why module-level `return` statements are a thing.

It looks like I was wrong about this not overriding anything, as this is [enabled by default when the `node` environment is set](https://github.com/eslint/eslint/blob/master/conf/environments.js#L90). That being said, this is the first time I've seen a project explicitly turn it off. I don't think this is correct, because `parserOptions` should be used to configure the parser for the environment rather than to disallow syntax. If the goal is to disallow module-level `return` statements I think a rule should be used instead (again, I haven't seen other projects do this, but it's definitely possible!).

Reply:
Please don't remove `globalReturn`.
Disallowing syntax in `parserOptions` may be bad, but it does no harm.

Reply:
Just want to confirm that I should fix the following new errors that result from this leaving this off:

```
$ npx eslint . --fix

/Users/kai/Code/mocha/.eleventy.js
  1:1   error  Use the function form of 'use strict'  strict
  3:18  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/.wallaby.js
  1:1   error  Use the function form of 'use strict'  strict
  3:18  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/browser-entry.js
    1:1   error  Use the function form of 'use strict'  strict
   41:26  error  Use the function form of 'use strict'  strict
   59:14  error  Use the function form of 'use strict'  strict
   77:1   error  Use the function form of 'use strict'  strict
   93:28  error  Use the function form of 'use strict'  strict
  105:20  error  Use the function form of 'use strict'  strict
  117:12  error  Use the function form of 'use strict'  strict
  127:15  error  Use the function form of 'use strict'  strict
  143:13  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/example/config/.mocharc.js
  1:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/index.js
  1:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/karma.conf.js
    1:1   error  Use the function form of 'use strict'  strict
   19:18  error  Use the function form of 'use strict'  strict
  168:1   error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/browser/growl.js
    1:1   error  Use the function form of 'use strict'  strict
   25:21  error  Use the function form of 'use strict'  strict
   41:18  error  Use the function form of 'use strict'  strict
   70:1   error  Use the function form of 'use strict'  strict
  106:1   error  Use the function form of 'use strict'  strict
  121:1   error  Use the function form of 'use strict'  strict
  166:1   error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/browser/progress.js
   1:1   error  Use the function form of 'use strict'  strict
  12:1   error  Use the function form of 'use strict'  strict
  26:27  error  Use the function form of 'use strict'  strict
  38:27  error  Use the function form of 'use strict'  strict
  50:31  error  Use the function form of 'use strict'  strict
  61:27  error  Use the function form of 'use strict'  strict
  72:29  error  Use the function form of 'use strict'  strict
  83:27  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/browser/tty.js
  1:1   error  Use the function form of 'use strict'  strict
  3:18  error  Use the function form of 'use strict'  strict
  7:25  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/cli/cli.js
   3:1   error  Use the function form of 'use strict'                          strict
  30:16  error  Wrap arrow function in a function with 'use strict' directive  strict

/Users/kai/Code/mocha/lib/cli/collect-files.js
   1:1   error  Use the function form of 'use strict'                          strict
  29:18  error  Wrap arrow function in a function with 'use strict' directive  strict

/Users/kai/Code/mocha/lib/cli/commands.js
  1:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/cli/config.js
   1:1   error  Use the function form of 'use strict'                          strict
  31:31  error  Use the function form of 'use strict'                          strict
  40:9   error  Use the function form of 'use strict'                          strict
  42:7   error  Use the function form of 'use strict'                          strict
  56:9   error  Use the function form of 'use strict'                          strict
  70:22  error  Use the function form of 'use strict'                          strict
  95:22  error  Wrap arrow function in a function with 'use strict' directive  strict

/Users/kai/Code/mocha/lib/cli/index.js
  1:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/cli/init.js
   1:1   error  Use the function form of 'use strict'  strict
  18:19  error  Use the function form of 'use strict'  strict
  24:19  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/cli/node-flags.js
   1:1   error  Use the function form of 'use strict'                          strict
  36:22  error  Wrap arrow function in a function with 'use strict' directive  strict
  66:29  error  Use the function form of 'use strict'                          strict
  77:28  error  Use the function form of 'use strict'                          strict

/Users/kai/Code/mocha/lib/cli/one-and-dones.js
   1:1   error  Use the function form of 'use strict'  strict
  21:18  error  Use the function form of 'use strict'  strict
  51:22  error  Use the function form of 'use strict'  strict
  58:21  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/cli/options.js
    1:1   error  Use the function form of 'use strict'                          strict
   66:5   error  Use the function form of 'use strict'                          strict
   73:7   error  Use the function form of 'use strict'                          strict
   89:11  error  Use the function form of 'use strict'                          strict
   99:15  error  Wrap arrow function in a function with 'use strict' directive  strict
  158:24  error  Use the function form of 'use strict'                          strict
  175:23  error  Wrap arrow function in a function with 'use strict' directive  strict
  219:16  error  Wrap arrow function in a function with 'use strict' directive  strict
  236:19  error  Wrap arrow function in a function with 'use strict' directive  strict
  280:21  error  Wrap arrow function in a function with 'use strict' directive  strict

/Users/kai/Code/mocha/lib/cli/run-helpers.js
    1:1   error  Use the function form of 'use strict'                          strict
   26:24  error  Use the function form of 'use strict'                          strict
   39:19  error  Use the function form of 'use strict'                          strict
   74:16  error  Use the function form of 'use strict'                          strict
   82:26  error  Wrap arrow function in a function with 'use strict' directive  strict
  104:21  error  Wrap arrow function in a function with 'use strict' directive  strict
  117:20  error  Use the function form of 'use strict'                          strict
  157:26  error  Wrap arrow function in a function with 'use strict' directive  strict

/Users/kai/Code/mocha/lib/cli/run-option-metadata.js
  1:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/cli/run.js
    1:1   error  Use the function form of 'use strict'  strict
   45:19  error  Use the function form of 'use strict'  strict
  302:19  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/cli/watch-run.js
    1:1   error  Use the function form of 'use strict'                          strict
   31:18  error  Wrap arrow function in a function with 'use strict' directive  strict
   87:24  error  Use the function form of 'use strict'                          strict
  140:25  error  Use the function form of 'use strict'                          strict
  157:20  error  Use the function form of 'use strict'                          strict
  172:20  error  Use the function form of 'use strict'                          strict
  181:20  error  Use the function form of 'use strict'                          strict
  189:19  error  Use the function form of 'use strict'                          strict

/Users/kai/Code/mocha/lib/context.js
   1:1   error  Use the function form of 'use strict'  strict
  16:1   error  Use the function form of 'use strict'  strict
  25:30  error  Use the function form of 'use strict'  strict
  40:29  error  Use the function form of 'use strict'  strict
  55:36  error  Use the function form of 'use strict'  strict
  70:26  error  Use the function form of 'use strict'  strict
  84:26  error  Use the function form of 'use strict'  strict
  95:29  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/errors.js
    1:1  error  Use the function form of 'use strict'  strict
   17:1  error  Use the function form of 'use strict'  strict
   32:1  error  Use the function form of 'use strict'  strict
   47:1  error  Use the function form of 'use strict'  strict
   61:1  error  Use the function form of 'use strict'  strict
   76:1  error  Use the function form of 'use strict'  strict
   89:1  error  Use the function form of 'use strict'  strict
  108:1  error  Use the function form of 'use strict'  strict
  124:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/growl.js
    1:1   error  Use the function form of 'use strict'  strict
   27:21  error  Use the function form of 'use strict'  strict
   44:18  error  Use the function form of 'use strict'  strict
   56:17  error  Use the function form of 'use strict'  strict
   98:1   error  Use the function form of 'use strict'  strict
  113:14  error  Use the function form of 'use strict'  strict
  129:28  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/hook.js
   1:1   error  Use the function form of 'use strict'  strict
  20:1   error  Use the function form of 'use strict'  strict
  38:24  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/interfaces/bdd.js
   1:1   error  Use the function form of 'use strict'  strict
  24:18  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/interfaces/common.js
   1:1   error  Use the function form of 'use strict'  strict
  15:18  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/interfaces/exports.js
   1:1   error  Use the function form of 'use strict'  strict
  22:18  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/interfaces/index.js
  1:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/interfaces/qunit.js
   1:1   error  Use the function form of 'use strict'  strict
  32:18  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/interfaces/tdd.js
   1:1   error  Use the function form of 'use strict'  strict
  32:18  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/mocha.js
    1:1   error  Use the function form of 'use strict'  strict
   93:1   error  Use the function form of 'use strict'  strict
  149:24  error  Use the function form of 'use strict'  strict
  167:27  error  Use the function form of 'use strict'  strict
  188:28  error  Use the function form of 'use strict'  strict
  252:22  error  Use the function form of 'use strict'  strict
  305:29  error  Use the function form of 'use strict'  strict
  325:20  error  Use the function form of 'use strict'  strict
  343:31  error  Use the function form of 'use strict'  strict
  361:25  error  Use the function form of 'use strict'  strict
  402:24  error  Use the function form of 'use strict'  strict
  425:26  error  Use the function form of 'use strict'  strict
  440:31  error  Use the function form of 'use strict'  strict
  457:30  error  Use the function form of 'use strict'  strict
  471:29  error  Use the function form of 'use strict'  strict
  484:25  error  Use the function form of 'use strict'  strict
  533:26  error  Use the function form of 'use strict'  strict
  555:29  error  Use the function form of 'use strict'  strict
  572:25  error  Use the function form of 'use strict'  strict
  588:34  error  Use the function form of 'use strict'  strict
  606:31  error  Use the function form of 'use strict'  strict
  621:28  error  Use the function form of 'use strict'  strict
  636:24  error  Use the function form of 'use strict'  strict
  665:27  error  Use the function form of 'use strict'  strict
  684:27  error  Use the function form of 'use strict'  strict
  706:24  error  Use the function form of 'use strict'  strict
  720:34  error  Use the function form of 'use strict'  strict
  736:29  error  Use the function form of 'use strict'  strict
  748:34  error  Use the function form of 'use strict'  strict
  762:33  error  Use the function form of 'use strict'  strict
  779:25  error  Use the function form of 'use strict'  strict
  793:30  error  Use the function form of 'use strict'  strict
  807:33  error  Use the function form of 'use strict'  strict
  847:23  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/pending.js
   1:1  error  Use the function form of 'use strict'  strict
  10:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/base.js
    1:1   error  Use the function form of 'use strict'  strict
  105:30  error  Use the function form of 'use strict'  strict
  131:9   error  Use the function form of 'use strict'  strict
  135:9   error  Use the function form of 'use strict'  strict
  139:15  error  Use the function form of 'use strict'  strict
  143:20  error  Use the function form of 'use strict'  strict
  147:7   error  Use the function form of 'use strict'  strict
  157:36  error  Use the function form of 'use strict'  strict
  166:1   error  Use the function form of 'use strict'  strict
  184:44  error  Use the function form of 'use strict'  strict
  209:16  error  Use the function form of 'use strict'  strict
  297:1   error  Use the function form of 'use strict'  strict
  337:27  error  Use the function form of 'use strict'  strict
  379:1   error  Use the function form of 'use strict'  strict
  392:1   error  Use the function form of 'use strict'  strict
  429:1   error  Use the function form of 'use strict'  strict
  472:1   error  Use the function form of 'use strict'  strict
  495:1   error  Use the function form of 'use strict'  strict
  517:1   error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/doc.js
   1:1  error  Use the function form of 'use strict'  strict
  33:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/dot.js
   1:1  error  Use the function form of 'use strict'  strict
  34:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/html.js
    1:1   error  Use the function form of 'use strict'  strict
   59:1   error  Use the function form of 'use strict'  strict
  255:1   error  Use the function form of 'use strict'  strict
  276:27  error  Use the function form of 'use strict'  strict
  285:26  error  Use the function form of 'use strict'  strict
  295:32  error  Use the function form of 'use strict'  strict
  312:1   error  Use the function form of 'use strict'  strict
  321:1   error  Use the function form of 'use strict'  strict
  345:1   error  Use the function form of 'use strict'  strict
  358:1   error  Use the function form of 'use strict'  strict
  371:1   error  Use the function form of 'use strict'  strict
  382:1   error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/index.js
  1:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/json-stream.js
   1:1  error  Use the function form of 'use strict'  strict
  32:1  error  Use the function form of 'use strict'  strict
  69:1  error  Use the function form of 'use strict'  strict
  81:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/json.js
    1:1  error  Use the function form of 'use strict'  strict
   33:1  error  Use the function form of 'use strict'  strict
   81:1  error  Use the function form of 'use strict'  strict
  103:1  error  Use the function form of 'use strict'  strict
  127:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/landing.js
   1:1  error  Use the function form of 'use strict'  strict
  54:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/list.js
   1:1  error  Use the function form of 'use strict'  strict
  37:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/markdown.js
   1:1  error  Use the function form of 'use strict'  strict
  39:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/min.js
   1:1  error  Use the function form of 'use strict'  strict
  34:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/nyan.js
    1:1   error  Use the function form of 'use strict'  strict
   34:1   error  Use the function form of 'use strict'  strict
   86:26  error  Use the function form of 'use strict'  strict
  101:36  error  Use the function form of 'use strict'  strict
  124:35  error  Use the function form of 'use strict'  strict
  143:33  error  Use the function form of 'use strict'  strict
  160:33  error  Use the function form of 'use strict'  strict
  196:26  error  Use the function form of 'use strict'  strict
  215:30  error  Use the function form of 'use strict'  strict
  226:32  error  Use the function form of 'use strict'  strict
  236:36  error  Use the function form of 'use strict'  strict
  258:32  error  Use the function form of 'use strict'  strict
  272:1   error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/progress.js
   1:1  error  Use the function form of 'use strict'  strict
  40:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/spec.js
   1:1  error  Use the function form of 'use strict'  strict
  37:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/tap.js
    1:1   error  Use the function form of 'use strict'  strict
   37:1   error  Use the function form of 'use strict'  strict
   91:1   error  Use the function form of 'use strict'  strict
  102:1   error  Use the function form of 'use strict'  strict
  116:1   error  Use the function form of 'use strict'  strict
  142:1   error  Use the function form of 'use strict'  strict
  149:38  error  Use the function form of 'use strict'  strict
  157:35  error  Use the function form of 'use strict'  strict
  168:35  error  Use the function form of 'use strict'  strict
  179:38  error  Use the function form of 'use strict'  strict
  191:35  error  Use the function form of 'use strict'  strict
  201:39  error  Use the function form of 'use strict'  strict
  221:1   error  Use the function form of 'use strict'  strict
  254:1   error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/reporters/xunit.js
    1:1   error  Use the function form of 'use strict'  strict
   46:1   error  Use the function form of 'use strict'  strict
  124:24  error  Use the function form of 'use strict'  strict
  139:25  error  Use the function form of 'use strict'  strict
  154:24  error  Use the function form of 'use strict'  strict
  198:1   error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/runnable.js
    1:1   error  Use the function form of 'use strict'  strict
   30:1   error  Use the function form of 'use strict'  strict
   71:30  error  Use the function form of 'use strict'  strict
  103:27  error  Use the function form of 'use strict'  strict
  122:37  error  Use the function form of 'use strict'  strict
  137:27  error  Use the function form of 'use strict'  strict
  147:32  error  Use the function form of 'use strict'  strict
  156:31  error  Use the function form of 'use strict'  strict
  165:31  error  Use the function form of 'use strict'  strict
  174:30  error  Use the function form of 'use strict'  strict
  186:35  error  Use the function form of 'use strict'  strict
  201:32  error  Use the function form of 'use strict'  strict
  212:32  error  Use the function form of 'use strict'  strict
  221:35  error  Use the function form of 'use strict'  strict
  231:30  error  Use the function form of 'use strict'  strict
  255:35  error  Use the function form of 'use strict'  strict
  278:30  error  Use the function form of 'use strict'  strict
  291:26  error  Use the function form of 'use strict'  strict
  455:36  error  Use the function form of 'use strict'  strict
  494:27  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/runner.js
     1:1   error  Use the function form of 'use strict'  strict
   128:1   error  Use the function form of 'use strict'  strict
   171:25  error  Use the function form of 'use strict'  strict
   188:30  error  Use the function form of 'use strict'  strict
   211:32  error  Use the function form of 'use strict'  strict
   233:28  error  Use the function form of 'use strict'  strict
   247:33  error  Use the function form of 'use strict'  strict
   286:25  error  Use the function form of 'use strict'  strict
   328:29  error  Use the function form of 'use strict'  strict
   354:25  error  Use the function form of 'use strict'  strict
   441:26  error  Use the function form of 'use strict'  strict
   474:27  error  Use the function form of 'use strict'  strict
   486:29  error  Use the function form of 'use strict'  strict
   498:28  error  Use the function form of 'use strict'  strict
   514:28  error  Use the function form of 'use strict'  strict
   554:29  error  Use the function form of 'use strict'  strict
   712:1   error  Use the function form of 'use strict'  strict
   723:29  error  Use the function form of 'use strict'  strict
   799:29  error  Use the function form of 'use strict'  strict
   870:32  error  Use the function form of 'use strict'  strict
   884:24  error  Use the function form of 'use strict'  strict
   949:26  error  Use the function form of 'use strict'  strict
   964:1   error  Use the function form of 'use strict'  strict
  1007:1   error  Use the function form of 'use strict'  strict
  1019:1   error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/stats-collector.js
   1:1  error  Use the function form of 'use strict'  strict
  41:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/suite.js
    1:1   error  Use the function form of 'use strict'  strict
   29:16  error  Use the function form of 'use strict'  strict
   48:1   error  Use the function form of 'use strict'  strict
  101:25  error  Use the function form of 'use strict'  strict
  122:27  error  Use the function form of 'use strict'  strict
  144:27  error  Use the function form of 'use strict'  strict
  160:34  error  Use the function form of 'use strict'  strict
  176:24  error  Use the function form of 'use strict'  strict
  195:24  error  Use the function form of 'use strict'  strict
  209:29  error  Use the function form of 'use strict'  strict
  220:31  error  Use the function form of 'use strict'  strict
  240:29  error  Use the function form of 'use strict'  strict
  264:28  error  Use the function form of 'use strict'  strict
  288:30  error  Use the function form of 'use strict'  strict
  312:29  error  Use the function form of 'use strict'  strict
  335:28  error  Use the function form of 'use strict'  strict
  355:27  error  Use the function form of 'use strict'  strict
  375:29  error  Use the function form of 'use strict'  strict
  387:29  error  Use the function form of 'use strict'  strict
  405:25  error  Use the function form of 'use strict'  strict
  421:28  error  Use the function form of 'use strict'  strict
  433:23  error  Use the function form of 'use strict'  strict
  445:27  error  Use the function form of 'use strict'  strict
  461:30  error  Use the function form of 'use strict'  strict
  492:35  error  Use the function form of 'use strict'  strict
  502:34  error  Use the function form of 'use strict'  strict
  510:28  error  Use the function form of 'use strict'  strict
  524:35  error  Use the function form of 'use strict'  strict
  634:11  error  Use the function form of 'use strict'  strict
  637:11  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/test.js
   1:1   error  Use the function form of 'use strict'  strict
  19:1   error  Use the function form of 'use strict'  strict
  39:24  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/lib/utils.js
    1:1   error  Use the function form of 'use strict'  strict
   41:18  error  Use the function form of 'use strict'  strict
   52:20  error  Use the function form of 'use strict'  strict
   63:16  error  Use the function form of 'use strict'  strict
   76:17  error  Use the function form of 'use strict'  strict
  105:22  error  Use the function form of 'use strict'  strict
  128:1   error  Use the function form of 'use strict'  strict
  152:25  error  Use the function form of 'use strict'  strict
  173:1   error  Use the function form of 'use strict'  strict
  207:28  error  Use the function form of 'use strict'  strict
  236:21  error  Use the function form of 'use strict'  strict
  283:1   error  Use the function form of 'use strict'  strict
  379:24  error  Use the function form of 'use strict'  strict
  456:1   error  Use the function form of 'use strict'  strict
  478:1   error  Use the function form of 'use strict'  strict
  498:23  error  Use the function form of 'use strict'  strict
  586:1   error  Use the function form of 'use strict'  strict
  603:21  error  Use the function form of 'use strict'  strict
  619:16  error  Use the function form of 'use strict'  strict
  634:28  error  Use the function form of 'use strict'  strict
  699:21  error  Use the function form of 'use strict'  strict
  714:17  error  Use the function form of 'use strict'  strict
  733:18  error  Use the function form of 'use strict'  strict
  752:18  error  Use the function form of 'use strict'  strict
  782:20  error  Use the function form of 'use strict'  strict
  792:16  error  Use the function form of 'use strict'  strict
  809:21  error  Use the function form of 'use strict'  strict
  828:27  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/package-scripts.js
   1:1  error  Use the function form of 'use strict'  strict
  12:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/scripts/dedefine.js
   1:1   error  Use the function form of 'use strict'  strict
  10:1   error  Use the function form of 'use strict'  strict
  17:18  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/scripts/linkify-changelog.js
  1:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/scripts/markdown-magic.config.js
   1:1   error  Use the function form of 'use strict'  strict
  20:10  error  Use the function form of 'use strict'  strict
  41:8   error  Use the function form of 'use strict'  strict
  59:9   error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/scripts/netlify-headers.js
   1:1   error  Use the function form of 'use strict'  strict
  14:1   error  Use the function form of 'use strict'  strict
  33:10  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/scripts/package-json-cullify.js
   1:1   error  Use the function form of 'use strict'  strict
  12:18  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/scripts/update-authors.js
   6:1   error  Use the function form of 'use strict'  strict
  46:15  error  Use the function form of 'use strict'  strict
  60:16  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/assertions.js
  1:1   error  Use the function form of 'use strict'  strict
  3:32  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/browser-specific/setup.js
  1:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/bundle/amd.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:48  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/color.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:26  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/compiler-globbing.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:39  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/config.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  10:20  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/deprecate.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:34  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/diffs.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  15:1   error  Use the function form of 'use strict'  strict
  54:1   error  Use the function form of 'use strict'  strict
  71:19  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/duplicate-arguments.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  5:64  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/events.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:25  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/file-utils.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  10:24  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/glob.spec.js
    1:1   error  Use the function form of 'use strict'  strict
    8:22  error  Use the function form of 'use strict'  strict
  179:32  error  Use the function form of 'use strict'  strict
  185:29  error  Use the function form of 'use strict'  strict
  190:21  error  Use the function form of 'use strict'  strict
  197:1   error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/helpers.js
    1:1   error  Use the function form of 'use strict'  strict
   37:13  error  Use the function form of 'use strict'  strict
   74:17  error  Use the function form of 'use strict'  strict
  160:17  error  Use the function form of 'use strict'  strict
  171:1   error  Use the function form of 'use strict'  strict
  184:1   error  Use the function form of 'use strict'  strict
  188:1   error  Use the function form of 'use strict'  strict
  214:1   error  Use the function form of 'use strict'  strict
  236:1   error  Use the function form of 'use strict'  strict
  245:1   error  Use the function form of 'use strict'  strict
  267:1   error  Use the function form of 'use strict'  strict
  301:1   error  Use the function form of 'use strict'  strict
  308:1   error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/hook-err.spec.js
    1:1   error  Use the function form of 'use strict'  strict
    9:33  error  Use the function form of 'use strict'  strict
  278:1   error  Use the function form of 'use strict'  strict
  288:1   error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/hooks.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  9:19  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/init.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  9:26  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/invalid-arguments.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  5:31  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/multiple-done.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:38  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/no-diff.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:21  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/only.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:21  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/allowUncaught.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  8:30  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/asyncOnly.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:26  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/bail.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:20  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/compilers.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:25  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/delay.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:21  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/exit.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:20  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/extension.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:25  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/file.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  8:20  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/forbidOnly.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  8:27  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/forbidPending.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  8:30  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/grep.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  9:20  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/help.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:20  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/ignore.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  8:22  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/invert.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  5:22  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/listInterfaces.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  8:31  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/listReporters.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  8:30  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/node-flags.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  5:24  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/opts.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  9:20  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/reporter-option.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:31  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/retries.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:23  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/sort.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:20  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/timeout.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:23  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/ui.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:18  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/options/watch.spec.js
    1:1   error  Use the function form of 'use strict'  strict
    8:21  error  Use the function form of 'use strict'  strict
  293:1   error  Use the function form of 'use strict'  strict
  320:1   error  Use the function form of 'use strict'  strict
  329:1   error  Use the function form of 'use strict'  strict
  339:1   error  Use the function form of 'use strict'  strict
  345:1   error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/pending.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  12:21  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/regression.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:25  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/reporters.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  11:23  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/retries.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  8:21  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/suite.spec.js
   1:1   error  Use the function form of 'use strict'  strict
   6:33  error  Use the function form of 'use strict'  strict
  25:41  error  Use the function form of 'use strict'  strict
  39:39  error  Use the function form of 'use strict'  strict
  53:37  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/timeout.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:28  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/integration/uncaught.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:33  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/interfaces/bdd.spec.js
   1:1   error  Use the function form of 'use strict'  strict
   3:32  error  Use the function form of 'use strict'  strict
  17:32  error  Use the function form of 'use strict'  strict
  26:23  error  Use the function form of 'use strict'  strict
  36:27  error  Use the function form of 'use strict'  strict
  44:27  error  Use the function form of 'use strict'  strict
  50:53  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/interfaces/exports.spec.js
   1:1   error  Use the function form of 'use strict'  strict
   6:11  error  Use the function form of 'use strict'  strict
  10:10  error  Use the function form of 'use strict'  strict
  25:17  error  Use the function form of 'use strict'  strict
  29:16  error  Use the function form of 'use strict'  strict
  33:55  error  Use the function form of 'use strict'  strict
  39:66  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/interfaces/qunit.spec.js
   1:1   error  Use the function form of 'use strict'  strict
   5:20  error  Use the function form of 'use strict'  strict
   9:26  error  Use the function form of 'use strict'  strict
  18:17  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/interfaces/tdd.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  3:29  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/jsapi/index.js
   1:1   error  Use the function form of 'use strict'  strict
  24:11  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/node-unit/cli/config.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:24  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/node-unit/cli/node-flags.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  10:24  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/node-unit/cli/options.spec.js
   1:1   error  Use the function form of 'use strict'                          strict
  11:26  error  Wrap arrow function in a function with 'use strict' directive  strict
  34:21  error  Use the function form of 'use strict'                          strict

/Users/kai/Code/mocha/test/node-unit/cli/run-helpers.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:31  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/node-unit/cli/run.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:21  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/node-unit/mocha.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:19  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/node-unit/stack-trace-filter.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:32  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/only/bdd-require.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  9:42  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/only/bdd.spec.js
    1:1   error  Use the function form of 'use strict'  strict
    3:58  error  Use the function form of 'use strict'  strict
   15:39  error  Use the function form of 'use strict'  strict
   29:57  error  Use the function form of 'use strict'  strict
   43:58  error  Use the function form of 'use strict'  strict
   62:41  error  Use the function form of 'use strict'  strict
   82:62  error  Use the function form of 'use strict'  strict
  100:63  error  Use the function form of 'use strict'  strict
  117:54  error  Use the function form of 'use strict'  strict
  121:54  error  Use the function form of 'use strict'  strict
  125:45  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/only/global/bdd.spec.js
   1:1   error  Use the function form of 'use strict'  strict
   4:58  error  Use the function form of 'use strict'  strict
   8:57  error  Use the function form of 'use strict'  strict
  12:57  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/only/global/qunit.spec.js
   1:1   error  Use the function form of 'use strict'  strict
   4:62  error  Use the function form of 'use strict'  strict
   8:61  error  Use the function form of 'use strict'  strict
  12:61  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/only/global/tdd.spec.js
   1:1   error  Use the function form of 'use strict'  strict
   4:60  error  Use the function form of 'use strict'  strict
   8:59  error  Use the function form of 'use strict'  strict
  12:59  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/only/qunit.spec.js
   1:1   error  Use the function form of 'use strict'  strict
   4:56  error  Use the function form of 'use strict'  strict
   8:56  error  Use the function form of 'use strict'  strict
  12:47  error  Use the function form of 'use strict'  strict
  18:34  error  Use the function form of 'use strict'  strict
  21:35  error  Use the function form of 'use strict'  strict
  24:74  error  Use the function form of 'use strict'  strict
  31:33  error  Use the function form of 'use strict'  strict
  35:33  error  Use the function form of 'use strict'  strict
  39:33  error  Use the function form of 'use strict'  strict
  46:34  error  Use the function form of 'use strict'  strict
  50:34  error  Use the function form of 'use strict'  strict
  54:34  error  Use the function form of 'use strict'  strict
  61:37  error  Use the function form of 'use strict'  strict
  65:42  error  Use the function form of 'use strict'  strict
  69:37  error  Use the function form of 'use strict'  strict
  73:42  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/only/tdd.spec.js
    1:1   error  Use the function form of 'use strict'  strict
    3:55  error  Use the function form of 'use strict'  strict
   15:36  error  Use the function form of 'use strict'  strict
   29:54  error  Use the function form of 'use strict'  strict
   43:55  error  Use the function form of 'use strict'  strict
   62:38  error  Use the function form of 'use strict'  strict
   82:59  error  Use the function form of 'use strict'  strict
  100:60  error  Use the function form of 'use strict'  strict
  117:56  error  Use the function form of 'use strict'  strict
  121:56  error  Use the function form of 'use strict'  strict
  125:47  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/opts/opts.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:20  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/base.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  14:27  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/doc.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  16:26  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/dot.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  19:26  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/helpers.js
    1:1  error  Use the function form of 'use strict'  strict
   20:1  error  Use the function form of 'use strict'  strict
   51:1  error  Use the function form of 'use strict'  strict
  146:1  error  Use the function form of 'use strict'  strict
  155:1  error  Use the function form of 'use strict'  strict
  163:1  error  Use the function form of 'use strict'  strict
  189:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/json-stream.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  19:34  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/json.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  9:27  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/landing.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  21:30  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/list.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  20:27  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/markdown.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  16:31  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/min.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  15:26  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/nyan.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  19:27  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/progress.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  17:31  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/spec.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  18:27  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/tap.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  18:26  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/reporters/xunit.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  27:28  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/require/a.js
  1:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/require/c.js
  1:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/require/require.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  3:26  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/sanity/sanity.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  5:48  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/setup.js
  1:1  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/context.spec.js
   1:1   error  Use the function form of 'use strict'  strict
   3:21  error  Use the function form of 'use strict'  strict
  34:30  error  Use the function form of 'use strict'  strict
  78:21  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/duration.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  3:23  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/errors.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  5:20  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/globals.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  3:26  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/grep.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  5:19  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/hook-async.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  3:19  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/hook-sync-nested.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  3:20  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/hook-sync.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  3:20  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/hook-timeout.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  3:8   error  Use the function form of 'use strict'  strict
  8:19  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/mocha.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:19  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/overspecified-async.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  3:58  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/required-tokens.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  7:37  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/root.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  5:8   error  Use the function form of 'use strict'  strict
  9:18  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/runnable.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  9:33  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/runner.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  20:20  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/suite.spec.js
    1:1   error  Use the function form of 'use strict'  strict
    9:1   error  Use the function form of 'use strict'  strict
   14:19  error  Use the function form of 'use strict'  strict
  643:18  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/test.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:18  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/throw.spec.js
   1:1   error  Use the function form of 'use strict'  strict
  13:32  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/timeout.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  3:22  error  Use the function form of 'use strict'  strict

/Users/kai/Code/mocha/test/unit/utils.spec.js
  1:1   error  Use the function form of 'use strict'  strict
  6:23  error  Use the function form of 'use strict'  strict

✖ 688 problems (688 errors, 0 warnings)
```

Reply:
Ha, I see what's going on here. This is currently set to `no`, which ESLint doesn't recognize 🤷‍♂. So removing it should make no difference.

Reply:
[Here](https://github.com/eslint/eslint/blob/master/lib/rules/strict.js#L108) is the relevant line in the code. It's getting passed `"no"`, which is truthy.

Reply:
ok, thank you, you convinced me. We remove this line.